server/models/CollectionJob.js

```diff
@@ -9,7 +9,8 @@ module.exports = function (sequelize, DataTypes) {
             id: {
                 type: DataTypes.STRING,
                 allowNull: false,
-                primaryKey: true
+                primaryKey: true,
+                autoIncrement: true
```

Comment:
Think this shouldn't be here. I keep seeing this in recent PRs. Very popular piece of code 😅

Reply:
Ok I just removed it.